java/src/main/java/com/saucelabs/simplesauce/examples/BasicUsage.java

```diff
@@ -11,6 +11,6 @@ public static void main(String[] argv){
 
         sauce.getWebDriver().get("https://www.saucedemo.com/");
 
-        sauce.stop();
+        sauce.stop(true);
```

Comment:
We don't want to supply an actual test watcher object from the test here?

Reply:
Maybe? I'm more used to after hooks than test watchers... What would it look like?
Would the stop code live in the test watcher?
TestNG gives you `ITestResult` for this...